src/main/java/Mona/Mona.java

```diff
@@ -0,0 +1,88 @@
+package mona;
```

Comment:
Perhaps you can divide your classes into multiple packages to accommodate the larger number of classes?